src/applications/disability-benefits/new-686/tests/config/reportDivorce.unit.spec.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import {
+  DefinitionTester,
+  fillData,
+  selectRadio,
+} from 'platform/testing/unit/schemaform-utils.jsx';
+import formConfig from '../../config/form';
+
+describe('686 report a divorce', () => {
+  const {
+    schema,
+    uiSchema,
+  } = formConfig.chapters.reportDivorce.pages.formerSpouseDetails;
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        schema={schema}
+        uiSchema={uiSchema}
+        definitions={formConfig.defaultDefinitions}
+      />,
+    );
+    expect(form.find('input').length).to.equal(8);
+    form.unmount();
+  });
+
+  it('should not submit an empty form', () => {
+    const onSubmit = sinon.spy();
+    const form = mount(
+      <DefinitionTester
+        schema={schema}
+        uiSchema={uiSchema}
+        definitions={formConfig.defaultDefinitions}
+        onSubmit={onSubmit}
+      />,
+    );
+    form.find('form').simulate('submit');
+    expect(form.find('.usa-input-error').length).to.equal(6);
+    expect(onSubmit.called).to.be.false;
+    form.unmount();
+  });
+
+  it('should submit a valid form without an explanation of annullment or voided marriage', () => {
+    const onSubmit = sinon.spy();
+    const form = mount(
+      <DefinitionTester
+        schema={schema}
+        uiSchema={uiSchema}
+        definitions={formConfig.defaultDefinitions}
+        onSubmit={onSubmit}
+      />,
+    );
+    // spouse name
+    fillData(form, 'input#root_formerSpouseName_first', 'John');
+    fillData(form, 'input#root_formerSpouseName_last', 'Doe');
+    // date of divorce
+    const monthDropdown = form.find('select#root_dateOfDivorceMonth');
+    const dayDropdown = form.find('select#root_dateOfDivorceDay');
+    monthDropdown.simulate('change', {
```

Comment:
I wonder if we should to an RFC and request that the platform team add a helper method for changing drop downs?